appsignals-tests/images/aws-sdk/aws-sdk-v1/src/main/java/com/amazon/sampleapp/App.java

```diff
@@ -501,4 +519,246 @@ private static void setupS3() {
+          ObjectNode jsonResponse = objectMapper.createObjectNode();
+          jsonResponse.put("nextToken", "testToken");
+          ArrayNode memoryContents = objectMapper.createArrayNode();
+          jsonResponse.set("memoryContents", memoryContents);
+
+          res.status(mainStatus);
+          res.type("application/json");
+          return jsonResponse;
+        });
+
+    get(
+        "/bedrockagent/getknowledgeBase/:knowledgeBaseId",
+        (req, res) -> {
+          setMainStatus(200);
+          String knowledgeBaseId = req.params(":knowledgeBaseId");
+          try {
+            GetKnowledgeBaseRequest request =
+                new GetKnowledgeBaseRequest().withKnowledgeBaseId(knowledgeBaseId);
+            bedrockAgentClient.getKnowledgeBase(request);
+          } catch (Exception e) {
+
+          }
+          return "";
+        });
+
+    get(
+        "/bedrockruntime/error",
+        (req, res) -> {
+          setMainStatus(404);
+          var errorClient =
+              AmazonBedrockRuntimeClient.builder()
+                  .withCredentials(CREDENTIALS_PROVIDER)
+                  .withEndpointConfiguration(
+                      new EndpointConfiguration(
+                          "http://error.test:8080", Regions.US_WEST_2.getName()))
+                  .build();
+
+          String modelId = "anthropic.claude-v2";
+          InvokeModelRequest invokeModelRequest =
+              new InvokeModelRequest()
+                  .withModelId(modelId)
+                  .withBody(
+                      StandardCharsets.UTF_8.encode(
+                          "{\"prompt\":\"Hello, world!\",\"temperature\":0.7,\"top_p\":0.9,\"max_tokens_to_sample\":100}\n"));
+          errorClient.invokeModel(invokeModelRequest);
+          return "";
+        });
+
+    get(
+        "/bedrockagent/fault",
+        (req, res) -> {
+          setMainStatus(500);
+          var faultClient =
+              AWSBedrockAgentClient.builder()
+                  .withCredentials(CREDENTIALS_PROVIDER)
+                  .withEndpointConfiguration(
+                      new EndpointConfiguration(
+                          "http://fault.test:8080", Regions.US_WEST_2.getName()))
+                  .build();
+
+          try {
+            GetDataSourceRequest request =
+                new GetDataSourceRequest()
+                    .withDataSourceId("nonExistDatasourceId")
+                    .withKnowledgeBaseId("nonExistKnowledgeBaseId");
+            faultClient.getDataSource(request);
+          } catch (Exception e) {
+
+          }
+          return "";
+        });
+    get(
+        "/bedrockagent/getagent/:agentId",
+        (req, res) -> {
+          setMainStatus(200);
+          String testAgentId = req.params(":agentId");
+          try {
+            GetAgentRequest request = new GetAgentRequest().withAgentId(testAgentId);
+            bedrockAgentClient.getAgent(request);
+          } catch (Exception e) {
+
```

Comment:
we need to add the log or error stack for troubleshooting incase unexpected exceptions occurred
Eg, 
```
logger.info("Exception Caught in Sample App");
e.printStackTrace();
```